.goreleaser.yaml

```diff
@@ -0,0 +1,67 @@
+before:
+  hooks:
+  - go mod tidy
```

Comment:
Why is this needed?

Reply:
Not strictly needed, but this was part of the default file generated by `goreleaser init`. I think it makes sense to prune unused dependencies before the build.